tools/flang1/flang1exe/lowersym.c

```diff
@@ -1221,13 +1221,96 @@ lower_pointer_init(void)
   }
 } /* lower_pointer_init */
 
+static void
+replace_fval_in_params(int entry)
+{
+  int fval, fvalsame, newdsc, newdscsame, newarg, newargsame;
+  int params, narg, i;
+
+  fval = FVALG(entry);
+  fvalsame = LOWER_SYMBOL_REPLACE(fval);
+  newdsc = NEWDSCG(fval);
+  newarg = NEWARGG(fval);
+  newdscsame = NEWDSCG(fvalsame);
+  newargsame = NEWARGG(fvalsame);
+  params = DPDSCG(entry);
+  narg = PARAMCTG(entry);
+  for (i = 0; i < narg; i++) {
+    int arg = aux.dpdsc_base[params + i];
+    if (arg != 0 && arg == newarg) {
+      aux.dpdsc_base[params + i] = newargsame;
+      continue;
+    }
+    if (arg != 0 && arg == newdsc) {
+      aux.dpdsc_base[params + i] = newdscsame;
+      continue;
+    }
+  }
+}
+
+static void
+replace_sptr_in_ast(int sptr)
+{
+  int newsptr, ast;
+
+  if (sptr == 0 || sptr == NOSYM) {
+    return;
+  }
+  newsptr = LOWER_SYMBOL_REPLACE(sptr);
+  if (newsptr == 0 || newsptr == NOSYM){
```

Comment:
There should be a space before the opening brace.

Reply:
Space added (and expression re-written).